tests/test-ospec.js

```diff
@@ -304,6 +304,22 @@ o.spec("ospec", function() {
 					done()
 				})
 			})
+			/*eslint-disable no-eval*/
+			try {
+				eval("(()=>{})()")
```

Comment:
Have you actually run this in IE or PhantomJS? (PhantomJS works just as well for these purposes.) Also, [please re-read my comment thread.](https://github.com/MithrilJS/ospec/pull/2#discussion_r336796240)

Reply:
> Have you actually run this in IE or PhantomJS? (PhantomJS works just as well for these purposes.) Also, [please re-read my comment thread.](https://github.com/MithrilJS/ospec/pull/2#discussion_r336796240)

No, I haven't test them on IE.
Reason I did that way is because similar code is already in the that test file.
See https://github.com/MithrilJS/ospec/blob/master/tests/test-ospec.js#L717

I tried to run tests with PhantomJS, but I couldn't get them working even without any changes. May be it's just my unfamiliarity with how PhantomJS works.

Reply:
Try loading `https://polyfill.io/v3/polyfill.min.js?flags=gated%2Calways&features=Promise` (a promise polyfill) before `tests/test-ospec.js`, and that should make the test failures disappear.

Reply:
Sorry, can you show me how to run a test with PhantomJS?
Do I have to write a separate JS file for that?
I've tried `phantomjs ospec.js tests/test-ospec.js`, but that doesn't do anything.

Reply:
Edit: try creating this wrapper in the project root and executing it directly (without arguments). It's been a while since I've really toyed around with PhantomJS.

```js
var o = require("./ospec.js")
require("./tests/test-ospec.js")
o.run()
```

Reply:
Thanks for that. Adding promise polyfill did the trick.
There were couple of existing tests that was failing.
I was managed to fix one, but not sure how to fix other one so I created #8.